src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -23,16 +22,14 @@
 using Microsoft.VisualStudio.R.Package.Repl;
 
 namespace Microsoft.R.Components.InteractiveWorkflow.Implementation {
-    public sealed class RInteractiveWorkflow : IRInteractiveWorkflow {
+    public sealed class RInteractiveWorkflow : IRInteractiveWorkflowVisual {
```

Comment:
Right. But currently `IRInteractiveWorkflow` has `Task<IInteractiveWindowVisualComponent> GetOrCreateVisualComponentAsync`.  Where do we move it.